--- sa-1.2.2/Makefile.in~	2004-04-02 20:17:31.000000000 +0200
+++ sa-1.2.2/Makefile.in	2004-11-28 02:50:50.000000000 +0100
@@ -69,7 +69,7 @@
 
 $(LIB_NAME): $(LIB_OBJS)
 	@$(LIBTOOL) --mode=link $(CC) -o $(LIB_NAME) $(LIB_OBJS) -rpath $(libdir) \
-	    -version-info `$(SHTOOL) version -l txt -d libtool VERSION`
+	    -version-info `$(SHTOOL) version -l txt -d libtool VERSION` $(LIBS)
 
 $(TST_NAME): $(TST_OBJS) $(LIB_NAME)
 	@$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o $(TST_NAME) $(TST_OBJS) $(LIB_NAME) $(LIBS)
@@ -96,7 +96,7 @@
 	$(SHTOOL) install -c -m 755 sa-config $(DESTDIR)$(bindir)/
 	$(SHTOOL) install -c -m 644 sa.h $(DESTDIR)$(includedir)/
 	$(SHTOOL) install -c -m 644 sa.3 $(DESTDIR)$(mandir)/man3/
-	@$(LIBTOOL) --mode=install $(SHTOOL) install -c -m 644 libsa.la $(DESTDIR)$(libdir)/
+	@$(LIBTOOL) --mode=install $(SHTOOL) install -c -m 755 libsa.la $(DESTDIR)$(libdir)/
 
 uninstall:
 	@$(LIBTOOL) --mode=uninstall $(RM) $(DESTDIR)$(libdir)/libsa.la
